frontend/integration-tests/tests/metalkube/dashboard.scenario.ts

```diff
@@ -0,0 +1,69 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+const execSync = require('child_process').execSync;
+
+import { appHost } from '../../protractor.conf';
+import * as dashboardView from '../../views/metalkube/dashboards.view';
+
+async function compareCounter(elem, expectedValue) {
+  const displayedValue = Number(await dashboardView.getTextIfPresent(elem, '0'));
+  expect(displayedValue).toEqual(expectedValue);
+}
+
+describe('Inventory card', () => {
+  beforeAll(async() => {
+    await browser.get(`${appHost}/dashboards`);
+    await dashboardView.isLoaded();
+    // wait until the counters in the inventory card show up
+    await browser.wait(
+      until.or(
+        until.presenceOf(dashboardView.inventoryNodesUpCounter),
+        until.presenceOf(dashboardView.inventoryNodesDownCounter)));
+    await browser.wait(
+      until.or(
+        until.presenceOf(dashboardView.inventoryHostsUpCounter),
+        until.presenceOf(dashboardView.inventoryHostsDownCounter)));
+  });
+
+  it('Node count is displayed', async() => {
+    // get the number of ready and not ready nodes from the CLI
+    let readyNodes = 0;
+    const output = execSync('oc get nodes -o json', { encoding: 'utf-8' });
+    const nodes = JSON.parse(output);
+    let node;
```

Comment:
You could do:
```
nodes.items.forEach((node) => {
  node.status.conditions.forEach((condition) => {
        if (condition.reason === 'KubeletReady' && condition.status === 'True') {
          readyNodes++;
        }
    });
});
```
to avoid using helper variables and make the iterations easier to read.